src/Command/Processor/KeyPrefixProcessor.php

```diff
@@ -31,142 +31,154 @@ class KeyPrefixProcessor implements ProcessorInterface
     public function __construct($prefix)
     {
         $this->prefix = $prefix;
-        $this->commands = array(
+
+        $prefixFirst = static::class . '::first';
```

Comment:
We could also have here now the callables as:
```php
$prefixFirst = static function (CommandInterface $command, $prefix) {
    static::first($command, $prefix);
};
```
that can be upgraded in PHP 8.1+ to:
```php
$prefixFirst = static::first(...);
```